src/framework/xr/xr-view.js

```diff
@@ -324,6 +324,14 @@ class XrView extends EventHandler {
         return this._viewInvOffMat;
     }
 
+    /**
+     * @type {Mat4}
+     * @ignore
+     */
+    get viewInvMat() {
```

Comment:
Camera component exposes viewMatrix only, I'd be keen to not expose this at all.

Reply:
Why is per-eye view matrix needed?

Reply:
It technically isnt but the model viewer gets the inverse matrix using `const mat = xr.views.list[0]._viewInvMat;`
but ive replaced it with `const mat = xr.camera.camera.viewMatrix.clone().invert();`